test/e2e/utilization.e2e.ts

```diff
@@ -93,8 +93,8 @@ test.describe('System utilization', () => {
     await expectRowVisible(page.getByRole('table'), {
       Silo: 'all-zeros',
       CPU: '0',
-      Memory: '0 GiB',
-      Storage: '0 TiB',
+      Memory: '0 B',
+      Storage: '0 B',
```

Comment:
A thing to talk about, for example. Is `B` fine here? Do we want, instead, some minimum (`GiB`?)